packages/smart-wallet/src/offerWatcher.js

```diff
@@ -10,7 +10,7 @@ import {
 import { AmountShape } from '@agoric/ertp/src/typeGuards.js';
 import { deeplyFulfilledObject, objectMap } from '@agoric/internal';
 
-import { UNPUBLISHED_RESULT } from './offers.js';
+import { UNPUBLISHED_RESULT } from './offers.ts';
```

Comment:
This is a runtime import from `.ts`. Not sure we're ok with that in the smart wallet?